rs/sns/governance/src/lib.rs

```diff
@@ -121,12 +126,35 @@ fn validate_required_field<'a, Inner>(
         .ok_or_else(|| format!("The {} field must be populated.", field_name))
 }
 
-/// Return an Err whose inner value describes (in detail) what is wrong with a
-/// field value, and where within some (Protocol Buffers message) struct.
-fn field_err(field_name: &str, field_value: impl Debug, defect: &str) -> Result<(), String> {
+/// Return an Err whose inner value describes (in detail) what is wrong with a field value (should
```

Comment:
Dropped this remark, as it was referring to the responsibility to bound the `field_value` argument that no longer is on the caller (since the function now bounds the byte size of that thing before printing it).